src/protocol.h

```diff
@@ -29,6 +27,16 @@
 class CMessageHeader
 {
 public:
+    enum {
+        MESSAGE_START_SIZE = 4,
```

Comment:
Though it looks like we use this kind of constant declaration in many places, and so does leveldb:
```
arith_uint256.h:    enum { WIDTH=BITS/32 };
chain.h:    enum { nMedianTimeSpan=11 };
leveldb/db/skiplist.h:  enum { kMaxHeight = 12 };
leveldb/helpers/memenv/memenv.cc:  enum { kBlockSize = 8 * 1024 };
...
```
So I suppose it's somewhat acceptable in some cases?